pkg/service/admission/endpoint_admission.go

```diff
@@ -0,0 +1,115 @@
+package admission
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+	oadmission "github.com/openshift/origin/pkg/cmd/server/admission"
+
+	kadmission "k8s.io/kubernetes/pkg/admission"
+	kapi "k8s.io/kubernetes/pkg/api"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const RestrictedEndpointsPluginName = "RestrictedEndpointsAdmission"
+
+func init() {
+	kadmission.RegisterPlugin("RestrictedEndpointsAdmission", func(client clientset.Interface, config io.Reader) (kadmission.Interface, error) {
```

Comment:
I don't know. Possibly we don't, but all of the other explicitly-created admission controller types do it too.